example/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+PROJECT(mciExample CXX)
+
+FIND_PACKAGE(ITK REQUIRED)
```

Comment:
Add `COMPONENTS MorphologicalContourInterpolation` and other modules required (try `ITK/Utilities/Maintenance/WhatModulesITK.py` so the MorphologicalContourInterpolation include directories will be added when build as a Remote module.